make/examples.mk

```diff
@@ -3,10 +3,6 @@
 
 export KUBERNETES_VERSION := v1.27.5
 
-export CLUSTERCTL_VERSION := $(shell clusterctl version -o short 2>/dev/null)
-export CAPA_VERSION := $(shell cd hack/third-party/capa && go list -m -f '{{ .Version }}' sigs.k8s.io/cluster-api-provider-aws/v2)
-export CAPX_VERSION := $(shell cd hack/third-party/capx && go list -m -f '{{ .Version }}' github.com/nutanix-cloud-native/cluster-api-provider-nutanix)
-
```

Comment:
These were duplicates from `clusterctl.mk` so I removed them from here and consolidated there.